src/DETHRACE/common/world.c

```diff
@@ -2798,6 +2798,22 @@ void LoadTrack(char* pFile_name, tTrack_spec* pTrack_spec, tRace_info* pRace_inf
 
         if (!strcmp(s, "none") || !strcmp(s, "NONE") || !strcmp(s, "0") || !strcmp(s, "1")) {
             pRace_info->material_modifiers[i].skid_mark_material = NULL;
+#if defined(DETHRACE_FIX_BUGS)
+            /*
+             * Fix OG bug, where "INDUSTB1.TXT" and "INDUSTB2.TXT" tracks don't
+             * define any skidmark materials, preventing skidmarks from being
+             * displayed in those races. As a workaround, use skidmark names
+             * for material #0 as found in "INDUSTB3.TXT" (another variation of
+             * this track design).
+             */
+            if (i == 0 &&
```

Comment:
Are there tracks where the first skid material is `NULL`, but have valid skid materials for later indices?
I'm asking because it would be nice to avoid hard coding track file names.

Can we set the default skid mark on some condition?

e.g. add this at the end of the for loop.
```c
if (pRace_info->material_modifiers[0].skid_mark_material == NULL) {
    LOG_WARN("Track %s has no valid first skid mark material, setting the default one", pRace_info->track_file_name);
    LoadSinglePixelmap(&gTrack_storage_space, "SKIDMARK.PIX");
    material = LoadSingleMaterial(&gTrack_storage_space, "SKIDMARK.MAT");
    pRace_info->material_modifiers[i].skid_mark_material = material;
}
```

Reply:
Checked all the tracks for CARMA and SPLATPAK. The only tracks that don't set skid material (`INDUSTB1.TXT`, `INDUSTB2.TXT` and `NETINDB.TXT`), lack it for all the material indices. So I went ahead and modified your suggestion, so that it only applies the workaround when all the skid materials are missing.